plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/handler/ScaleDeploymentHandler.java

```diff
@@ -90,42 +183,135 @@ protected IStatus run(IProgressMonitor monitor) {
 				
 			}.schedule();
 		}
-		
+	}
+
+	public boolean hasConfigChangeTrigger(IDeploymentConfig dc) {
+		return dc.getTriggers().stream()
+				.anyMatch(t -> IDeploymentConfigChangeTrigger.class.isAssignableFrom(t.getClass()));
```

Comment:
These triggers should have a type retrievable from the generic type that does not require comparison to the interrface